specs/011-pipeline-destructive-step-safety/quickstart.md

```
@@ -0,0 +1,61 @@
+
+```bash
+poetry run pytest tests/core/domain/test_pipeline_steps.py -v -k "destructive"
+```
+
+Expected: 9 tests pass (7 new gating tests + 1 property test + 1 updated integration test).
+
+### 2. Verify engine-level gating
+
+When an ingest pipeline encounters an error (e.g., storage write failure), the engine logs:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)